docs/src/pages/components/tooltips/AccessibilityTooltips.tsx

```diff
@@ -1,18 +1,14 @@
 import * as React from 'react';
-import DeleteIcon from '@material-ui/icons/Delete';
 import Button from '@material-ui/core/Button';
-import IconButton from '@material-ui/core/IconButton';
 import Tooltip from '@material-ui/core/Tooltip';
 
 export default function AccessibilityTooltips() {
   return (
     <div>
       <Tooltip title="Delete">
-        <IconButton>
-          <DeleteIcon />
-        </IconButton>
+        <Button>Delete</Button>
```

Comment:
The new example doesn't make much sense. We're repeating the content in the title and I'm fairly certain the `title` acts as a description here as well because the label comes from content.

Reply:
Oh right, I have overlooked this aspect. Would using `<DeleteIcon />` be enough? (without `IconButton`)?

Reply:
Why would you add a tooltip for an icon?

Reply:
Ok, well my only objective was to have as little code as possible and have the two demos as close as possible to have the eye catch the different of the tooltip props. If no further improvement is possible in this direction, we can ignore my change on the demo!